kaiax/reward/spec.go

```diff
@@ -135,8 +136,8 @@ type AccumulatedRewardsResponse struct {
 
 func (spec RewardSpec) ToAccumulatedResponse(firstHeader, lastHeader *types.Header) *AccumulatedRewardsResponse {
 	return &AccumulatedRewardsResponse{
-		FirstBlockTime: firstHeader.Time.String(),
-		LastBlockTime:  lastHeader.Time.String(),
+		FirstBlockTime: time.Unix(firstHeader.Time.Int64(), 0).UTC().String(),
```

Comment:
Can we make them integer type? it's more natural to return a unix timestamp as a number.

Reply:
I misunderstood before. Changed to int64. https://github.com/kaiachain/kaia/pull/249/commits/c4be6434c817d4ff95055d65932c68c17697a588